libcontainerd/remote/client.go

```diff
@@ -13,6 +13,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/opencontainers/go-digest"
+
 	"github.com/containerd/containerd"
```

Comment:
nit: looks like this ended up in a separate import group.


This commit should probably be combined with the containerd update (as without it, that one won't compile?)

Reply:
Fixed this while doing the rebase